internal/pkg/aws/cloudwatchlogs/cloudwatchlogs.go

```diff
@@ -101,15 +101,16 @@ func (c *CloudWatchLogs) logStreams(logGroup string, logStreamLimit int, logStre
 			break
 		}
 	}
+	logStreamNames = truncateStreams(logStreamLimit, logStreamNames)
 
-	return truncateStreams(logStreamLimit, logStreamNames), nil
+	// Make sure that the streams are returned with the latest events last so they're printed to terminal in chronological order.
+	return reverseStreams(logStreamNames), nil
```

Comment:
Hmm I think I would expect last events to be first for logs 🤔

Reply:
I wrote a long digression on why this is necessary, but I guess though it actually doesn't matter. Later on, in `LogEvents()`, we sort all the logs by event time so the order of streams doesn't matter, just that they are the most recent N streams chronologically.